### PR TITLE
Fixes #36656 - Pin @adobe/css-tools to 4.2.0

### PR DESCRIPTION
**Pull request by sjha4**

### CODE REVIEW

**Comment by evgeni** [comment]
is that something that needs packaging, or shall we add it to the exclude list?

**Comment by ofedoren** [comment]
Why exactly do we need to pin the lib instead of update node version in CI?..

**Comment by jeremylenz** [comment]
> Why exactly do we need to pin the lib instead of update node version in CI?..

We're stuck on node 12 downstream for the time being, as I understand it. We had [previously upgraded node](https://github.com/Katello/katello/pull/10520) here in Github Actions and then had to revert it.

**Comment by ofedoren** [comment]
> 
> We're stuck on node 12 downstream for the time being

oof :disappointed: Okay then. At least there is a reason...

**Comment by sjha4** [comment]
Updated to add this as dev-dependency and added it exclude list..I didn't see it being packaged and package-lock shows dev: true for this in foreman and katello.

**Review by evgeni — DISMISSED** [review]

**Review by jeremylenz — DISMISSED** [review]
APJ :+1:

File: package.json [hunk 1]
```diff
@@ -69,6 +69,7 @@
     "webpack-dev-server-without-h2": "^2.11.8",
     "webpack-stats-plugin": "^0.1.5",
     "tabbable": "~5.2.0",
-    "sass": "~1.60.0"
+    "sass": "~1.60.0",
+    "@adobe/css-tools": "~4.2.0"
```
Comment (ekohl) [inline package.json]: Nit: I prefer things to remain sorted. We already have `@` dependencies on top.

Comment (sjha4) [reply package.json]: Updated..

**Review by jeremylenz — APPROVED** [review]

**Review by evgeni — APPROVED** [review]

**Pull request merged**